cmake/oss.cmake.in

```diff
@@ -352,3 +352,16 @@ INSTALL_COMMAND
   COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/obj/libv8_monolith.a <INSTALL_DIR>/lib/libv8_monolith.a
   COMMAND ${CMAKE_COMMAND} -E copy_directory <SOURCE_DIR>/include <INSTALL_DIR>/include
 )
+
+ExternalProject_Add(avro
+  URL https://storage.googleapis.com/vectorizedio-public/dependencies/avro-release-1.10.2.tar.gz
```

Comment:
for OSS, we use github, not google bucket, unless there isn't a github avro ( i thought all apache projs did)

let's move this to vtools.